### ISSUE TITLE
Investigate running Quarkus project generator without waiting for Java extension

### ISSUE THREAD

**Issue by datho7561** [redhat-developer/vscode-quarkus#323]
When you are in rootless mode (no folder open, just individual files), the command to generate a Quarkus project doesn't appear. It could be helpful to display the command in this mode, since generating the project doesn't depend on being in a particular folder.

**Comment by rgrunber** [comment]
FWIW, I was able to see the command to create a Quarkus project with no folder opened (from the command palette), but I did have to wait on vscode-microprofile + vscode-java to initialize.

**Comment by datho7561** [comment]
Yes, I also can confirm this behaviour. Should I close this issue then, and open another one for investigating running the generator without starting vscode-java?
